src/library/returnBook/ReturnBookControl.java

```diff
@@ -0,0 +1,95 @@
+package library.returnBook;
+import library.entities.Item;
+import library.entities.Library;
+import library.entities.Loan;
+
+public class ReturnBookControl {
+
+    private ReturnBookUI ui;
```

Comment:
#PR41 - D2: 
Please rename variable to ReturnItemUI as in the BRLS class diagram.

Reply:
https://github.com/elliotmitchum/ITC205/pull/41/commits/1774fdc9d206d8411b12829d60f152f49b04b6dc